package.json

```diff
@@ -46,76 +46,75 @@
     "to-style": "^1.3.3",
-    "victory-core": "^15.2.0"
+    "victory-core": "^31.1.0"
   },
   "peerDependencies": {
-    "prop-types": "^15.6.0",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "prop-types": "^15.6.2",
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.0.0-beta.42",
-    "@babel/core": "^7.0.0-beta.42",
-    "@babel/plugin-proposal-class-properties": "^7.0.0-beta.42",
-    "@babel/plugin-proposal-object-rest-spread": "^7.0.0-beta.42",
-    "@babel/polyfill": "^7.0.0-beta.42",
-    "@babel/preset-env": "^7.0.0-beta.42",
-    "@babel/preset-react": "^7.0.0-beta.42",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-eslint": "^8.2.2",
-    "babel-jest": "^22.1.0",
-    "babel-loader": "^8.0.0-beta.2",
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.2.2",
+    "@babel/plugin-proposal-class-properties": "^7.2.3",
+    "@babel/plugin-proposal-object-rest-spread": "^7.2.0",
+    "@babel/polyfill": "^7.2.5",
+    "@babel/preset-env": "^7.2.3",
+    "@babel/preset-react": "^7.0.0",
+    "babel-eslint": "^10.0.1",
+    "babel-jest": "^23.6.0",
+    "babel-loader": "^8.0.5",
     "babel-plugin-emotion": "^9.0.1",
     "builder": "^4.0.0",
-    "css-loader": "^0.28.11",
-    "enzyme": "^3.1.0",
-    "enzyme-adapter-react-16": "^1.0.1",
-    "enzyme-to-json": "3.1.2",
+    "css-loader": "^2.1.0",
+    "enzyme": "^3.8.0",
+    "enzyme-adapter-react-16": "^1.7.1",
+    "enzyme-to-json": "3.3.5",
     "eslint": "^4.19.0",
     "eslint-config-formidable": "^3.0.0",
     "eslint-config-prettier": "^2.9.0",
     "eslint-plugin-filenames": "^1.2.0",
     "eslint-plugin-import": "^2.9.0",
     "eslint-plugin-react": "^7.7.0",
-    "file-loader": "^1.1.11",
-    "husky": "^1.0.0-rc.1",
-    "jest": "^22.4.2",
+    "file-loader": "^3.0.1",
+    "husky": "^1.3.1",
+    "jest": "^23.6.0",
     "jest-serializer-enzyme": "^1.0.0",
-    "lint-staged": "^7.0.4",
-    "prettier": "^1.12.1",
-    "prop-types": "^15.5.10",
-    "raw-loader": "^0.5.1",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
-    "react-test-renderer": "^16.0.0",
-    "redbox-react": "1.5.0",
-    "rimraf": "^2.6.1",
-    "style-loader": "^0.20.3",
-    "surge": "^0.19.0",
-    "tslint": "^5.12.0",
+    "lint-staged": "^8.1.0",
+    "prettier": "^1.15.3",
+    "prop-types": "^15.6.2",
+    "raw-loader": "^1.0.0",
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0",
+    "react-test-renderer": "^16.7.0",
+    "redbox-react": "1.6.0",
+    "rimraf": "^2.6.3",
+    "style-loader": "^0.23.1",
+    "surge": "^0.20.1",
+    "tslint": "^5.12.1",
     "typescript": "^3.2.2",
-    "url-loader": "^1.0.1",
-    "webpack": "^4.26.1",
-    "webpack-cli": "^3.1.2",
-    "webpack-dev-server": "^3.1.1"
+    "url-loader": "^1.1.2",
+    "webpack": "^4.28.4",
+    "webpack-cli": "^3.2.1",
+    "webpack-dev-server": "^3.1.14"
   },
   "resolutions": {
-    "babel-core": "7.0.0-bridge.0",
+    "babel-core": "^7.0.0-0",
```

Comment:
It does need this as a resolution for the babel-jest dependency, so I'm leaving it here until that changes